.github/workflows/tracegen.yml

```diff
@@ -0,0 +1,60 @@
+name: tracegen
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+  pull_request:
+
+jobs:
+  build-dev:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Build tracegen
```

Comment:
Can't we use the Docker action?

https://github.com/marketplace/actions/build-and-push-docker-images#usage

Reply:
fixed.